incubator/sparkoperator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sparkoperator
 description: A Helm chart for Spark on Kubernetes operator
-version: 0.5.0
+version: 0.6.0
```

Comment:
Since `cleanupCrdsBeforeInstall` was removed, this is a breaking change. 

Even though this is an incubator chart – with major version zero – and therefore [not officially considered stable by semver](https://semver.org/#spec-item-4), we do our best in the helm/charts to inform users of version changes that would break on upgrades. Because of that, would you please add a note in the chart README per number 2 here https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning:

>Any breaking (backwards incompatible) changes to a chart should:
>1.  Bump the MAJOR version
>1. In the README, under a section called "Upgrading", describe the manual steps necessary to upgrade to the new (specified) MAJOR version

Something like this:

```md
## Upgrading

### To 0.6.0

#### Breaking changes

- `cleanupCrdsBeforeInstall` has been removed for Helm 3 compatibility. If you with to replicate this behavior before upgrading, do so manually (`kubectl delete CustomResourceDefinition sparkapplications.sparkoperator.k8s.io scheduledsparkapplications.sparkoperator.k8s.io`)

#### Non-breaking changes

- CRDs have moved to `/crds` directory, with template globbing, for both Helm 2 and 3 compatibility
- `app.kubernetes.io/name=sparkoperator` label is added to CRDs if installed at this version, for easier manual cleanup after chart deletion (`kubectl delete CustomResourceDefinition -l app.kubernetes.io/name=sparkoperator`)
```

This will require adding the labels to CRDs (see other comments below).